src/main/java/com/ibm/eventstreams/connect/mqsource/processor/JMSReader.java

```diff
@@ -0,0 +1,491 @@
+/**
+ * Copyright 2017, 2018, 2019 IBM Corporation
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.ibm.eventstreams.connect.mqsource.processor;
```

Comment:
Package change is unnecessary.

Reply:
@AndrewJSchofield Changed as per the suggestion.